code/index.html

```diff
@@ -1,22 +1,90 @@
 <!DOCTYPE html>
+
 <html>
+
   <head>
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
+  
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+<div class="webpage">
+    <header class="header-container">
+      <img class="logo" src="./logo.svg" />
+      <nav class="navigation">
+        <div class="desktop-menu">
+          <div class="navigation-item"><a>One</a></div>
+          <div class="navigation-item"><a>Two</a></div>
+          <div class="navigation-item"><a>Three</a></div>
+          <div class="navigation-item"><a>Four</a></div>
+        </div>
+        <img class="mobile-menu" src="./hamburgare.png">
+        </div>
+      </nav>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+  <section class="big-news-container">
```

Comment:
The `<header>` and this `<section>` are siblings, and should be on the same level of indentation